apps/wallet/src/shared/deepbook-client.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { getActiveNetworkSuiClient } from '_shared/sui-client';
+import { DeepBookClient } from '@mysten/deepbook';
+
+export async function getDeepbookClient(): Promise<DeepBookClient> {
+	const suiClient = await getActiveNetworkSuiClient();
```

Comment:
Probably it's a bit confusing, but this `getActiveNetworkSuiClient` should not be used from the UI context. For the ui we have the `useSuiClient` to access the sui client.

This `getActiveNetworkSuiClient` is used in the background service